src/perf/lib/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ target_include_directories(perflib PUBLIC ${CMAKE_CURRENT_LIST_DIR})
 if (MSVC)
     target_compile_options(perflib PUBLIC /wd4459)
 else()
-    target_compile_options(perflib PUBLIC -Wno-switch)
+    target_compile_options(perflib PUBLIC ${QUIC_WARNING_FLAGS})
```

Comment:
Just like below, warnings are already added with the target link libraries warning above. So we should just be able to completely remove this else branch now.

Reply:
I can double-check tomorrow, but I did see errors without these changes.

Reply:
I checked, and if i take this out, I get the following compile error:
```
In file included from /vagrant/src/perf/lib/PerfHelpers.h:32:
/vagrant/src/inc/msquic.hpp:239:11: error: field 'Handle' will be initialized after field 'CloseAllConnectionsOnDelete' [-Werror,-Wreorder]
        : Handle(nullptr), CloseAllConnectionsOnDelete(false) {
          ^
1 error generated.
```